### ISSUE TITLE
NUnitEngineException: Failed to start remote test agent.

### ISSUE THREAD

**Issue by un-tone** [Starcounter/Blending#74]
Moved from https://github.com/Starcounter/Blending/pull/73

I have a problem with starting tests. I'm getting the following issue:

```
Test Files
    C:\Projects\Blending\test\BlendingProvider.Tests\bin\Debug\BlendingProvider.Tests.dll

NUnit.Engine.NUnitEngineException: Failed to start remote test agent.
   в NUnit.Engine.Services.TestAgency.OnAgentExit(Object sender, EventArgs e)
   в System.Diagnostics.Process.OnExited()
   в System.Diagnostics.Process.RaiseOnExited()
   в System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   в System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   в System.Threading._ThreadPoolWaitOrTimerCallback.PerformWaitOrTimerCallback(Object state, Boolean timedOut)
```

It happens when running any of `*.Tests.dll` assemblies. If someone knows a solution for this, please let me know.

-----

Upgrading of NUnit packages to the last versions solves my problem. Maybe this is some case of incompatibility after upgrading projects to .NET 4.6.1 or something similar?

Do we have any impediments to upgrading all the packages?

**Comment by warpech** [comment]
@un-tone is it still valid?

**Comment by un-tone** [comment]
@warpech yes. I have such troubles only on one of three machines. I tried to clean repo and packages, but only upgrading helped me.

**Comment by warpech** [comment]
I think there is no problem to upgrade NUnit as long as the tests pass on TC.

@kegor can you pls do it?

**Comment by PatrykSzwer** [comment]
@un-tone please see the comment https://github.com/Starcounter/Blending/pull/81#issuecomment-366273313

**Comment by kegor** [comment]
Closed because fix was merged in #74.
